codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/DafnyLocalServiceTypeConversionProtocol.java

```diff
@@ -1255,12 +1349,24 @@ func Error_FromDafny(err $L.Error)(error) {
                 final var depService = context
                   .model()
                   .expectShape(dep, ServiceShape.class);
-                final var sdkId = depService.hasTrait(LocalServiceTrait.class)
```

Comment:
Using if else because I need to change sdk id like from "kms" to "ComAmazonawsKms"